vendor/bat-native-ledger/src/bat/ledger/internal/bat_state.cc

```diff
@@ -8,39 +8,40 @@
 
 #include "bat/ledger/internal/bat_state.h"
 #include "bat/ledger/internal/ledger_impl.h"
-#include "bat/ledger/internal/rapidjson_bat_helper.h"
+#include "bat/ledger/internal/state/client_state.h"
 
 namespace braveledger_bat_state {
 
 BatState::BatState(bat_ledger::LedgerImpl* ledger) :
       ledger_(ledger),
-      state_(new braveledger_bat_helper::CLIENT_STATE_ST()) {
+      state_(new ledger::ClientProperties()) {
 }
 
 BatState::~BatState() {
 }
 
 bool BatState::LoadState(const std::string& data) {
-  braveledger_bat_helper::CLIENT_STATE_ST state;
-  if (!braveledger_bat_helper::loadFromJson(&state, data.c_str())) {
+  ledger::ClientProperties state;
+  const ledger::ClientState client_state;
+  if (!client_state.FromJson(data, &state)) {
     BLOG(ledger_, ledger::LogLevel::LOG_ERROR) <<
       "Failed to load client state: " << data;
     return false;
   }
 
-  state_.reset(new braveledger_bat_helper::CLIENT_STATE_ST(state));
+  state_.reset(new ledger::ClientProperties(state));
 
   bool stateChanged = false;
 
   // fix timestamp ms to s conversion
-  if (std::to_string(state_->reconcileStamp_).length() > 10) {
-    state_->reconcileStamp_ = state_->reconcileStamp_ / 1000;
+  if (std::to_string(state_->reconcile_timestamp).length() > 10) {
+    state_->reconcile_timestamp = state_->reconcile_timestamp / 1000;
```

Comment:
I thought a tailing '_' on member vars was required by the standards? so `->reconcile_timestamp_`

Reply:
state_ is a data structure where underscore should not be used, thanks